pkg/controller/servicebindingrequest/binder.go

```diff
@@ -24,10 +26,15 @@ import (
 )
 
 var (
-	// containersPath logical path to find containers on supported objects
-	containersPath = []string{"spec", "template", "spec", "containers"}
-	// volumesPath logical path to find volumes on supported objects
-	volumesPath = []string{"spec", "template", "spec", "volumes"}
+	// containersPath has the logical path logical path
+	// to find containers on supported objects
+	// Used as []string{"spec", "template", "spec", "containers"}
+
+	pathToContainers = "spec.template.spec.containers"
+
+	// logical path to find volumes on supported objects
+	// used as []string{"spec", "template", "spec", "volumes"}
+	pathToVolumes = "spec.template.spec.volumes"
```

Comment:
Both `pathToContainers` and `pathToVolumes` are default values for two specific properties in `PodSpecPath`.

Perhaps `DefaultPodSpecContainersPath` and `DefaultPodSpecVolumesPath` would make it more clear?

Reply:
Agreed.

Reply:
Done 468641292ef4a7b1851ce33cbf1347f8606d812e